flapjack/commands.py

```diff
@@ -127,14 +127,14 @@ def ensure_runtime(remote, runtime, branch, subpaths=False):
     if ext.flatpak('info', '--show-commit', runtime, branch, code=True) != 0:
         if remote is None:
             remote_name, remote_type = find_remote_for_runtime(runtime, branch)
-            ext.flatpak('install', '--{}'.format(remote_type), remote_name,
-                        runtime, branch)
+            ext.flatpak('install', '--assumeyes', '--{}'.format(remote_type),
+                        remote_name, runtime, branch)
```

Comment:
Just an observation, not a blocker.

Do we want the user to not confirm that a runtime is gonna be installed from a random candidate remote? Myself, for example, have `eos-sdk` and `eos-sdk-nightly` providing similar runtimes.

Also, since this code path would only affect the initial setup and not the 99% of `flapjack build` runs, it's probably not annoying to disable `--assumeyes` here.

Another solution is to return the amount of possible candidates in `find_remote_for_runtime` and use `--assumeyes` if there's only one candidate.